Source/WebCore/svg/properties/SVGPropertyOwnerRegistry.h

```diff
@@ -29,8 +29,15 @@
 #include "SVGAnimatedPropertyPairAccessorImpl.h"
 #include "SVGPropertyAccessorImpl.h"
 #include "SVGPropertyRegistry.h"
+#include <type_traits>
 #include <wtf/HashMap.h>
 
+template<typename T>
+concept fastPropertyFromAttribute = requires(const T& t)
```

Comment:
I would call this something like `HasFastPropertyFromAttribute` to make it more consistent with our other meta functions.

Also, very neat!